### PR TITLE
Add vultr.com

### PR DESCRIPTION
**Pull request by jackreeds**
The site has a couple of `svg` elements which aren't supposed to be modified. And a white patch.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -157,6 +157,24 @@ body > #root > #wrapper > div[class^='c'] > #main-content > div:first-child
...
 
+*.vultr.com
+
+CSS
+.svg-banner-shape {
+    fill: var(--darkreader-neutral-background);
+}
+
+IGNORE INLINE STYLE
+.icon-ui*
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:168]: Or I've missed something important in the syntax of CSS, but this shouldn't work and especially not with our matching system. And will probably error dark reader into something like:
'.icon-ui*' is not a valid selector

Comment (jackreeds) [reply src/config/dynamic-theme-fixes.config:168]: I intended to tell darkreader to not touch `.icon-ui` at all, so I'm selecting all its children. What if it is changed to `.icon-ui *` (with a space in-between)?

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -157,6 +157,24 @@ body > #root > #wrapper > div[class^='c'] > #main-content > div:first-child
 
 ================================
 
+*.vultr.com
+
+CSS
+.svg-banner-shape {
+    fill: var(--darkreader-neutral-background);
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:164]: don't forget the `!important;`

Comment (jackreeds) [reply src/config/dynamic-theme-fixes.config]: Hmm, maybe it is a good idea to tell the auto-formatter to do this too.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config]: > maybe it is a good idea to tell the auto-formatter to do this too.

Not really, certain fixes will break when you add `!important;` and it's hard to migrate this false-positives

**Review by Gusted — APPROVED** [review]

**Pull request merged**